src/qt/forms/sendcoinsentry.ui

```diff
@@ -18,7 +18,7 @@
   </property>
   <widget class="QFrame" name="SendCoins">
    <property name="toolTip">
-    <string>This is a normal payment.</string>
+    <string>The amount of Bitcoin to send</string>
```

Comment:
Maybe this tooltip could be removed?

Reply:
Yes, either keep it here or in the code, not both. This generates a spurious translation message.

Reply:
I meant to remove this for the dynamic tooltip